src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj

```diff
@@ -9,8 +9,9 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.CSharp</RootNamespace>
     <AssemblyName>Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests</AssemblyName>
-    <TargetFramework>netstandard1.3</TargetFramework>
-    <RoslynProjectType>UnitTestPortable</RoslynProjectType>
+    <RoslynProjectType>UnitTest</RoslynProjectType>
+    <PackageTargetFallback>portable-net45</PackageTargetFallback>
```

Comment:
Ah, I hadn't realized we were still on an old enough version of xUnit for that to matter